conjur/read_config.py

```diff
@@ -39,20 +43,33 @@ class Config():
 
     _config = {}
 
+    # TODO update default config file location if user has env variable set
     def __init__(self, config_file=DEFAULT_CONFIG_FILE, netrc_file=DEFAULT_NETRC_FILE):
-        logging.info("Trying to get configuration from filesystem (%s)...", config_file)
+        if 'CONJURRC' in os.environ:
```

Comment:
Move hardcoded string 'CONJURRC' to const

Reply:
removed this functionality